src/integrationTest/java/uk/gov/hmcts/reform/sscs/controllers/ServiceHearingsControllerTest.java

```diff
@@ -259,15 +258,16 @@ void testPostRequestServiceHearingValues_missingCase() throws Exception {
     @DisplayName("When Authorization and Case ID valid should return the case name with a with 200 response code")
     @Test
     void testPostRequestServiceLinkedCases() throws Exception {
-        List<LinkedCase> linkedCases = new ArrayList<>();
         List<String> reasonsforLink = new ArrayList<>();
-        linkedCases.add(LinkedCase.builder()
-                            .caseReference(String.valueOf(CASE_ID_LINKED))
-                            .caseName(CASE_NAME)
-                            .reasonsForLink(reasonsforLink)
-                            .build());
-        ServiceLinkedCases model = ServiceLinkedCases.builder().linkedCases(linkedCases).build();
-        String json = asJsonString(model);
+        List<ServiceLinkedCases> serviceLinkedCases = new ArrayList<>();
+
+        serviceLinkedCases.add(ServiceLinkedCases.builder()
```

Comment:
minor: the formatting of these method calls is a bit too indented.. can you check your intelliJ settings?